python-modules/cis_notifications/cis_notifications/event.py

```diff
@@ -72,21 +72,21 @@ def send(self, notification):
 
         [return] Dictionary of status codes by publisher.
         """
-        if not self.access_token:
+        if not self.access_token_dict:
             self.access_token_dict = json.loads(self.secret_manager.secretmgr("az_access_token"))
             # Auth0 will return in how long the token expires. We want "when", so do that
             # Note that if we have no expiration, we'll make sure we're expired in the past already
-            self.access_token_dict["exp"] = int(self.access_token_dict.get("expires_in", -99999999999)) + time.time()
-            # 10s leeway
-            if int(self.access_token_dict["exp"]) < time.time() - 10:
-                logger.info("Access token has expired, refreshing")
-                authzero = self._get_authzero_client()
-                self.access_token_dict = authzero.exchange_for_access_token()
-                self.access_token_dict["exp"] = time.time() + int(self.access_token_dict("expires_in"), 60)
-                self.secret_manager.secretmgr_store("az_access_token", self.access_token_dict)
-            else:
-                logger.info("Re-using cached access token")
+            self.access_token_dict["exp"] = int(self.access_token_dict.get("exp", -99999999999))
+        # 10s leeway
+        if not self.access_token or int(self.access_token_dict["exp"]) < time.time() - 10:
+            logger.info("Access token has expired, refreshing")
+            authzero = self._get_authzero_client()
+            self.access_token_dict = authzero.exchange_for_access_token()
+            self.access_token_dict["exp"] = time.time() + int(self.access_token_dict("expires_in"), 60)
```

Comment:
seems wrong still - this seems more like it:
```suggestion
            self.access_token_dict["exp"] = time.time() + int(self.access_token_dict.get("expires_in", 60))
```